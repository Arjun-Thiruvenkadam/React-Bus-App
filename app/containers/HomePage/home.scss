.Home {
  color: #0f3460;
  font-family: 'Sacramento', cursive;
  font-size: x-large;
  font-weight: bold;
  padding-top: 10px;
  background-color: #9be3de;
}

.Home img {
  width: 30%;
}
