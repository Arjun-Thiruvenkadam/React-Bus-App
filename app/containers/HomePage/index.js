/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';
import Bus from '../../images/bus.png';
import CardsHolder from '../../components/CardsHolder/index';
import './home.scss';

export default function HomePage() {
  return (
    <div className="Home">
      <h1>
        <FormattedMessage {...messages.header} />
      </h1>
      <img src={Bus} alt="Bus" />
      <CardsHolder />
    </div>
  );
}
