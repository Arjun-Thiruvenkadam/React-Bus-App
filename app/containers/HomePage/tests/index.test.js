import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { MemoryRouter } from 'react-router';
import HomePage from '../index';

describe('<HomePage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <MemoryRouter>
        <IntlProvider locale="en">
          <HomePage />
        </IntlProvider>
      </MemoryRouter>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
