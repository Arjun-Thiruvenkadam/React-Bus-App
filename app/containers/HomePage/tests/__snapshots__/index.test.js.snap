// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<div
  class="Home"
>
  <h1>
    <span>
      Welcome to Aju's Bus
    </span>
  </h1>
  <img
    alt="Bus"
    src="IMAGE_MOCK"
  />
  <div
    class="CardsHolder"
  >
    <br />
    <h3>
      <span>
        Let's start your journey
      </span>
    </h3>
    <br />
    <div
      class="Card"
    >
      <img
        alt="Safety"
        src="IMAGE_MOCK"
      />
      <h3>
        Safety
      </h3>
      <p>
        <span>
          We provide only the best for our customers. We take extra measures to ensure your safe travel.
        </span>
      </p>
    </div>
    <div
      class="Card"
    >
      <img
        alt="Price"
        src="IMAGE_MOCK"
      />
      <h3>
        Price
      </h3>
      <p>
        <span>
          We always give you the lowest price with the best partner offers.
        </span>
      </p>
    </div>
    <br />
    <br />
    <a
      class="Navlink"
      href="/order"
    >
      <span>
        Book Tickets
      </span>
    </a>
    <br />
    <br />
  </div>
</div>
`;
